fdbclient/vexillographer/fdb.options

```diff
@@ -182,6 +182,8 @@ description is not currently required but encouraged.
     <Option name="transaction_include_port_in_address" code="505"
             description="Addresses returned by get_addresses_for_key include the port when enabled. As of api version 630, this option is enabled by default and setting this has no effect."
             defaultFor="23"/>
+    <Option name="disable_tracing" code="600"
```

Comment:
There's already an option called trace_enable, so I'm concerned those two concepts will get confused

Reply:
Yes, there's also `SERVER_REQUEST_TRACING` and a few other similar types of options. This is something I need to think about though. I'm not too familiar with `TRACE_ENABLE` but it's possible it can be subsumed by this feature.

This is an early draft of this PR too, more to come :)

Reply:
Np, sorry about the early drive-by review 😄 . Naming stuff is hard. Just wanted to mention it I guess

Reply:
The `trace_enable` network option refers to trace logging (e.g. the XML or JSON files we generate). We also have `server_request_tracing` which enables the `TransactionDebug` and `CommitDebug` events at various points in the request path on client and server and `log_transaction` which records a lot of details about what a transaction is doing into the trace logs on the client.

Reply:
Thanks AJ, I remember [TRACE_ENABLE](https://apple.github.io/foundationdb/api-general.html#setting-network-options-with-environment-variables) now.

I suppose `server_request_tracing` should stick around, because it looks like it enables very detailed logs of the transaction, whereas the idea for general request tracing is to record timing related info for every transaction but not necessarily ACTOR state at every step.

`trace_enable` is a network option, `server_request_tracing` is a transaction option, and what I'm adding (`disable_tracing`) is a database option... I agree this gets very confusing. At the very least I need to think of a better name for `disable_tracing`.

Reply:
I renamed `disable_tracing` to `transaction_trace_disable` to be more in line with the `trace_enable` option (which is a very different option, but having a similar structure for the name is at least good I think).